sktime/forecasting/model_selection/_split.py

```diff
@@ -673,8 +673,11 @@ class CutoffSplitter(BaseSplitter):
     which using the notation provided in :class:`BaseSplitter`,
     can be written as :math:`\{k_1,\ldots,k_n\}` for integer based indexing,
```

Comment:
minor note: due to our recent discussion where we agreed that duplicate time index elements should be possible, the indices should be tuples, not sets - correct?

Reply:
This specific line refers to cutoffs. At least in my mind it does not make any sense to repeat cutoff values. That would be a very strange cross-validation exercise.

Reply:
of course. The point is about whether, still, it is better to keep using tuples everywhere, for consistency.

Reply:
Now it's all tuples